### ISSUE TITLE
Whether Distributed Optimizer wrapper meets the requirements?

### ISSUE THREAD

**Issue by shjwudp** [mlcommons/training_policies#302]
Hi, there. I want to ask whether the current rules allow the use of distributed optimizer wrappers like hvd.DistributedOptimizer ?

I see that we have a [list of optimizers allowed by the competition](https://github.com/mlperf/training_policies/blob/master/training_rules.adoc#appendix-allowed-optimizers). But maybe the distributed optimizer wrapper is a special case? My reasons as follow:

1.  For most reference implementations, the optimizer packaging has no special effect on the algorithm, such as Mask-RCNN + torch.optim.SGD.
2. Distributed training communication framework is an important part of training. Many companies are also studying the efficiency optimization of the framework. Supporting distributed training framework wrapper is conducive to promoting the technical exchange of distributed communication.

Thanks

**Comment by bitfort** [comment]
SWG:

AI(Everyone who wants) Weigh in on this.

**Comment by ShriyaRishab** [comment]
Distributed optimizers are generally allowed now, closing.
